netbsd/Platform.h

```diff
@@ -70,4 +71,20 @@ static inline void Platform_getRelease(char** string) {
    *string = Generic_uname();
 }
 
+#define PLATFORM_LONG_OPTIONS
```

Comment:
Should maybe include a note there are none.

Reply:
Should be fixed in 0fd67f9

Reply:
Added a note in 4331107